ui/webui/src/components/storage/MountPointMapping.jsx

```diff
@@ -546,6 +548,18 @@ const MountPointMappingContent = ({ deviceData, partitioningData, dispatch, idPr
 
 export const MountPointMapping = ({ deviceData, diskSelection, partitioningData, dispatch, idPrefix, setIsFormValid, onAddErrorNotification, reusePartitioning, setReusePartitioning, stepNotification }) => {
     const [usedPartitioning, setUsedPartitioning] = useState(partitioningData?.path);
+    const requiredMountPoints = useMemo(() => {
+        const mounts = [
+            { value: "/boot", name: "boot" },
+            { value: "/", name: "root" },
+        ];
+
+        cockpit.spawn(["ls", "/sys/firmware/efi"], { err: "ignore" })
+                .then(() => { mounts.push({ value: "/boot/efi", name: "boot-efi" }) })
+                .catch(() => { console.log("not an EFI system") });
```

Comment:
I think this should use `onCritFail` or similar (@rvykydal could be able to help) so that if it fails, the error reporting dialog shows up. But I'm fine with adding this in a followup PR later/after Beta. :)

Reply:
Oops, this was just for testing, we don't need to log this, removed.